MapWindow.cpp

```diff
@@ -590,13 +590,29 @@ const char *MapWindow::look(uint16 x, uint16 y, bool show_prefix)
 }
 
 
-Obj *MapWindow::get_objAtCursor()
+Obj *MapWindow::get_objAtCursor(bool for_use /* = false */)
 {
+    uint16 tile_n = tmp_map_buf[(cursor_y+TMP_MAP_BORDER) * tmp_map_width + (cursor_x+TMP_MAP_BORDER)];
+    if(tile_n == 0) //black area
+        return NULL; // nothing to see here. ;)
 
- if(tmp_map_buf[(cursor_y+TMP_MAP_BORDER) * tmp_map_width + (cursor_x+TMP_MAP_BORDER)] == 0) //black area
-   return NULL; // nothing to see here. ;)
-
- return obj_manager->get_obj(WRAPPED_COORD(cur_x + cursor_x, cur_level), cur_y + cursor_y, cur_level,OBJ_SEARCH_TOP, OBJ_EXCLUDE_IGNORED);
+    Obj *obj = obj_manager->get_obj(WRAPPED_COORD(cur_x + cursor_x, cur_level), cur_y + cursor_y, cur_level,OBJ_SEARCH_TOP, OBJ_EXCLUDE_IGNORED);
+    // Savage Empire Create Object from Tile
+    if (for_use && game_type == NUVIE_GAME_SE && obj == NULL)
+    {
+        Script *script = game->get_script();
+        Uint16 obj_n = script->call_get_tile_to_object_mapping(tile_n);
+        printf("mapped obj #: %x\n", obj_n);
```

Comment:
/nit Could you remove this printf statement

Reply:
yes, of course